src/lib/analytics-sources.ts

```diff
@@ -114,3 +118,27 @@ export function validateScoopManifestFile(snykExecutablePath: string): boolean {
   }
   return false;
 }
+
+export function isHomebrew(): boolean {
+  const currentProcessPath = process.execPath;
+  const isHomebrewPath = currentProcessPath.includes('Cellar/snyk');
```

Comment:
I'd consider extending the match

```suggestion
  const isHomebrewPath = currentProcessPath.includes('/Cellar/snyk/');
```

Reply:
good idea - done. thanks